src/components/plots/linePlot.ts

```diff
@@ -154,33 +154,70 @@ export module Plot {
       return { data: data, pixelPoints: pixelPoints, selection: d3.selectAll(allElements) };
     }
 
+    /**
+     * Retrieves the closest PlotData to queryPoint.
+     *
+     * Lines implement an x-dominant notion of distance; points closest in x are
+     * tie-broken by y distance.
+     *
+     * @param {Point} queryPoint The point to which plot data should be compared
+     *
+     * @returns {PlotData} The PlotData closest to queryPoint
+     */
     public getClosestPlotData(queryPoint: Point): PlotData {
-      var closestDistanceSquared = Infinity;
-      var closestDatum: any;
-      var closestSelection: D3.Selection;
-      var closestPoint: Point;
+      var minXDist = Infinity;
+      var minYDist = Infinity;
+
+      var closest: {
+        datum: any;
+        pixelPoint: Point;
+        node: Node
```

Comment:
Brandon and I discussed this and we think we should be as specific as possible. In this case, I'll go with Element.